JM/CF/Scripts/5_Mission/CommunityFramework/Mission/MissionBase.c

```diff
@@ -1,20 +1,35 @@
 modded class MissionBase
 {
+	protected bool m_bLoaded = false;
+	
```

Comment:
This should be `m_cf_bLoaded` as someone else might add a variable with this generic name into mission base.

Reply:
Can't fix it right now. Some mods will have a compile error